.env

```diff
@@ -32,3 +32,4 @@ REACT_APP_LINKED_EVENTS_SYSTEM_DATA_SOURCE=helsinki
 REACT_APP_SHOW_ADMIN=true
 REACT_APP_SHOW_REGISTRATION=true
 REACT_APP_LOCALIZED_IMAGE=true
+REACT_APP_ENABLE_EXTERNAL_USER_EVENTS=true
```

Comment:
You could document REACT_APP_ENABLE_EXTERNAL_USER_EVENTS variable to README.md. 

Not that in Platta you need to map REACT_APP_ENABLE_EXTERNAL_USER_EVENTS environment variable in https://dev.azure.com/City-of-Helsinki/linkedevents/_git/linkedevents-pipelines?path=/devops/linkedcomponents-ui-build-config.yml and https://dev.azure.com/City-of-Helsinki/linkedevents/_git/linkedevents-pipelines?path=/devops/linkedcomponents-ui-config-map.yml

And then add environment variables in https://dev.azure.com/City-of-Helsinki/linkedevents/_library?itemType=VariableGroups&view=VariableGroupView&variableGroupId=2922&path=linkedcomponents-ui-development

Reply:
Added to README: https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/194/commits/4c7cc77025e8c0cac7dbe3cf13bb3572198bb9d3

PR for Platta changes: https://dev.azure.com/City-of-Helsinki/linkedevents/_git/linkedevents-pipelines/pullrequest/5229